skll/data/readers.py

```diff
@@ -308,11 +308,10 @@ def _parse_dataframe(self, df, id_col, label_col, features=None):
             The id column.
         label_col : str or None
             The label column.
-        features : list of dict or None
-            The features, if they already exist;
-            if not, then they will be extracted
-            from the data frame.
-            Defaults to None.
+        ignore_blanks : bool, optional
+            If True, remove lines that have any NaN
```

Comment:
```suggestion
            If ``True``, remove lines that have any NaN
```

Reply:
Should we say `NaN values" or "blank values". Also s/lines/rows since this is tabular data?